pkg/issue/issue.go

```diff
@@ -81,19 +82,22 @@ func getPolicyIssue(ctx context.Context, issues issues, owner, repo, policy, tit
 // Ensure ensures an issue exists and is open for the provided repo and
 // policy. If opening, re-opening, or pinging an issue, the provided text will
 // be included.
-func Ensure(ctx context.Context, c *github.Client, owner, repo, policy, text string) error {
-	return ensure(ctx, c, c.Issues, owner, repo, policy, text)
+func Ensure(ctx context.Context, c *github.Client, owner, repo, policy, text string, at time.Time) error {
+	return ensure(ctx, c, c.Issues, owner, repo, policy, text, at)
```

Comment:
Remove the `at` parameter, and all the changes to the enforce package. We shouldn't change the package contract for testability. Create a private gloabal function pointer variable `timeNow` and set it to `time.Now` in the init function. Then in the tests, override the timeNow variable to a closure that returns the desired time for the test.